### PR TITLE
Fix issues found during testing / preperation for the virtual season

### PR DESCRIPTION
**Pull request by Flova**
Based on #4 

Changes:
- Don't define topics via the json as this seems like a non-standard way of doing this. Instead reasonable defaults are selected and they can easily be remapped by the user. See IPM for a similar discussion.
- Make the json devices file path a parameter
- Check if the json is at the path and give a helpful error if not
- Fix issue where ms are parsed as nanoseconds
- Fix camera info handling (no latch, timestamp, dimensions directly taken from the image)

### CODE REVIEW

**Review by SammyRamone — APPROVED** [review]
looks good
thanks for fixing the bugs and improving the code quality

**Comment by ijnek** [comment]
@Flova Would you be able to rebase onto current main?

**Comment by Flova** [comment]
I did a merge instead of a rebase, because I was on my phone and it was easier that way.

**Comment by Flova** [comment]
Now that the CI is working I need to fix linter stuff ^^

**Comment by Flova** [comment]
@SammyRamone and @ijnek I addressed the feedback and fixed some linting. I am not able to merge it on this repo so feel free to merge it after approval.

**Comment by SammyRamone** [comment]
> @SammyRamone and @ijnek I addressed the feedback and fixed some linting. I am not able to merge it on this repo so feel free to merge it after approval.

looks fine for me, but I will not merge it without @ijnek okay :sweat_smile:

File: CMakeLists.txt
Comment (ijnek) [inline CMakeLists.txt]:
```suggestion
install(DIRECTORY launch resources
  DESTINATION share/${PROJECT_NAME})
```

File: README.md
Comment (ijnek) [inline README.md:55]: There's one more reference to ``hlvs_player.launch`` on line 15 in this file which needs to be updated.

**Review by ijnek — APPROVED** [review]
Once the last comments are addressed, the rest looks good. Thanks @Flova!

**Comment by Flova** [comment]
I addressed said issue. So we can merge it after the CI finished. Thanks for your review @ijnek !

**Comment by Flova** [comment]
CI finished successfully. Merge? @SammyRamone

**Comment by SammyRamone** [comment]
> CI finished successfully. Merge? @SammyRamone

I will let @ijnek do it. I'm not sure if normal merge or rebase is preferred for this repo.

**Comment by ijnek** [comment]
Thanks @Flova!

**Pull request merged**
